enhancements/machine-api/remediation-history.md

```diff
@@ -158,11 +213,13 @@ TBD
 
 - Using events
   
-  The UI already displays events (in its own tab), but they aren't stored for very long, which will make it harder to see which nodes
-  are having recurring issues.
+  - The UI already displays events (in its own generic events tab), but events only have a short TTL (configurable,
+    defaults to 3 hours in OpenShift), which makes it hard to see which nodes are having recurring issues over a longer duration.
+  - Parsing events and composing a remediation overview is harder on the UI than to just display the MHC status.
 
 
 - Using metrics
 
-  Metrics are a good tool for recording e.g. how often remediations are triggered and how long they take in average, but
+  - Metrics are a good tool for recording e.g. how often remediations are triggered and how long they take in average, but
   not so much for tracking single remediations in the desired detail.
+  - Parsing getting and parsing metrics, and composing a remediation overview from them, is harder on the UI than to just display the MHC status.
```

Comment:
I think there's a typo at the beginning of this sentence, might wanna double check it